smoke_tests/sqs_receive_message_test.go

```diff
@@ -268,7 +268,7 @@ func Test_ReceiveMessageV1_xml_with_attributes(t *testing.T) {
 	assert.Equal(t, 1, len(receiveMessageResponse.Result.Messages))
 	assert.Equal(t, "MyTestMessage", receiveMessageResponse.Result.Messages[0].Body)
 	assert.Equal(t, "ad4883a84ad41c79aa3a373698c0d4e9", receiveMessageResponse.Result.Messages[0].MD5OfBody)
-	assert.Equal(t, "", receiveMessageResponse.Result.Messages[0].MD5OfMessageAttributes)
+	assert.Equal(t, "ae8770938aee44bc548cf65ac377e3bf", receiveMessageResponse.Result.Messages[0].MD5OfMessageAttributes)
```

Comment:
xml/json both should have hash for message attribute if the message attributes is not empty.